cli/azd/pkg/input/testdata/Test_progressLogManyLines.md

```diff
@@ -0,0 +1,151 @@
+*header*
+
+`prefix`── title ───────────────────────
+
+
+
+
+
+
+
+`prefix`────────────────────────────────
+
+## Next state 
+
+*header*
+
+`prefix`── title ───────────────────────
+
+
+
+
+
+`prefix`line: 0
+
+`prefix`────────────────────────────────
+
+## Next state 
```

Comment:
@weikanglim @wbreza , hopefully these snaps are easier to read :)
I've chosen `.md` so we even have colors syntax for free

Reply:
Thank you! This looks much better!